scripts/update-anserini.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
Should we rename this to `update-anserini-index.sh`? Update anserini sounds like updating Anserini's code.